src/Symfony/Component/Form/Resources/translations/validators.vi.xlf

```diff
@@ -114,6 +114,26 @@
                 <source>Please provide a valid phone number.</source>
                 <target>Vui lòng cung cấp số điện thoại hợp lệ.</target>
             </trans-unit>
+            <trans-unit id="124">
+                <source>The checkbox has an invalid value.</source>
+                <target>Hộp kiểm có một giá trị không hợp lệ.</target>
+            </trans-unit>
+            <trans-unit id="125">
+                <source>Please enter a valid email address.</source>
+                <target>Vui lòng nhập địa chỉ Email hợp lệ.</target>
```

Comment:
Should it be "Email" or "email"?

Reply:
Hi, thank you for your question. "Email" is english. So it doesn't matter for Vietnamese to use "Email" or "email".

Reply:
Okey. I didnt know that. 

Thank you